src/styl/responsive/tablet-portrait.styl

```diff
@@ -47,5 +46,6 @@ $header-height= 50px
 
 #map-holder
   iframe
-    width: 100%
+    position: absolute
```

Comment:
Adding positioning this breaks the layout. On my local with `absolute` the element has no height and doesnt display.